apps/explorer/forms.py

```diff
@@ -79,6 +79,10 @@ class PixelSetExportForm(forms.Form):
     )
 
 
+class PixelSetSelectForm(PixelSetExportForm):
```

Comment:
Why is it necessary?

Reply:
It's not required. I choose to declare a new form in case we need to adapt it in the future.

Reply:
Do we still need the other one then?